tests/unit/test_testing_farm.py

```diff
@@ -677,35 +678,46 @@ def test_payload(
 
 
 @pytest.mark.parametrize(
-    ("fmf_url," "fmf_ref," "result_url," "result_ref"),
+    ("fmf_url," "fmf_ref," "result_url," "result_ref," "merge_pr_in_ci"),
     [
-        (
+        (  # custom tests and specified ref
             "https://github.com/mmuzila/test",
             "main",
             "https://github.com/mmuzila/test",
             "main",
+            True,
         ),
-        (
+        (  # defaulting to the tests in repo, also merging
             None,
             None,
             "https://github.com/packit/packit",
             "feb41e5",
+            True,
         ),
-        (
+        (  # specifying only ref and merging
```

Comment:
I've almost forgotten, is this a valid use-case? It basically says that tests are in a different branch/tag/commit, but in the same repository; I think that in this case, it should be considered as a custom tests-scenario and we should not pass the `merge_sha`.

---

Although I cannot imagine use-case for tests from different _ref_ and in the same repository at the same time.

Reply:
+1, such merge wouldn't make sense and could easily screw things up

Reply:
> Although I cannot imagine use-case for tests from different ref and in the same repository at the same time.

don't underestimate, people are very creative :)

Reply:
OK, I'm going to update the `custom_fmf` meaning then… To make sure no merging happens with custom ref **∨**  url for fmf

Reply:
Looks like no change is required, when `fmf_url` is not set, the `fmf_ref` is ignored, both before and after this PR.